cmapPy/math/tests/test_modz.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import cmapPy.pandasGEXpress.setup_GCToo_logger as setup_logger
+import logging
+import pandas as pd
+import sys
+import cmapPy.math.modz as modz
+
+
+logger = logging.getLogger(setup_logger.LOGGER_NAME)
+
+test_mat = pd.DataFrame({'A':[1,2,3], 'B': [2,8,6], 'C': [6,8,9]})
+
+class TestModz(unittest.TestCase):
+    def test_calculate_weights(self):
+        raw_weights, weights = modz.calculate_weights(test_mat.corr())
+        self.assertTrue(len(weights == 3))
+        self.assertTrue(raw_weights.tolist() == [0.8183, 0.7202, 0.8838])
```

Comment:
If you did the rounding for easier testing, consider https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_array_almost_equal.html